app/Http/Controllers/Api/ApiV1Controller.php

```diff
@@ -4494,4 +4494,95 @@ public function statusUnpin(Request $request, $id)
 
         return $this->json($status);
     }
+
+    /**
+     * GET /api/v1/notifications
+     *
+     * @return array
+     */
+    public function accountNotificationsMarkAsRead(Request $request)
+    {
+        /* abort_if(! $request->user(), 403);
+        abort_unless($request->user()->tokenCan('write'), 403);*/
+
+        $user = $request->user();
+        $pid = $user->profile_id;
+        $id = $request->input('id');
+
+        if (empty($id)) {
```

Comment:
This needs to be in [another method](https://docs.joinmastodon.org/methods/notifications/#clear) to follow the Mastodon API

Reply:
OKay. I will apply